common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -99,7 +99,9 @@ public static void generateMaterialBlocks() {
                         .setData(ProviderType.BLOCKSTATE, NonNullBiConsumer.noop())
                         .setData(ProviderType.LANG, NonNullBiConsumer.noop())
                         .setData(ProviderType.LOOT, NonNullBiConsumer.noop())
-                        //.tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.MINEABLE_WITH_PICKAXE)
+                        .tag(BlockTags.BEACON_BASE_BLOCKS)
+                        // TODO: Add #forge:storage_blocks tag
```

Comment:
merge it after you remove these lines and re-run datagen

Reply:
Remove all 3 of the Green lines?